### PR TITLE
fix(FE/logs): enable horizontal scroll in logs context

### LINKED ISSUE #3400: LOGS: [FE] Horizontal scroll in in-context logs
There is no horizontal scroll in incontext logs.

### PR DESCRIPTION
**Pull request by CKanishka**
Fixes #3400 

**Description**

The content of a log in the Logs Context section was not fully visible and was getting truncated, so adding a horizontal scroll in order for the content to be visible on scrolling. 

**Changes Overview**

https://github.com/SigNoz/signoz/assets/30779692/311db868-45e2-4a30-bc59-fe333a318d3d

### CODE REVIEW

File: frontend/src/components/Logs/RawLogView/styles.ts [hunk 1]
```diff
@@ -35,19 +35,23 @@ interface RawLogContentProps {
...
 	font-family: Fira Code, monospace;
 	font-weight: 300;
 
-	overflow: hidden;
-	text-overflow: ellipsis;
-	display: -webkit-box;
-	-webkit-line-clamp: ${(props): number => props.linesPerRow};
-	line-clamp: ${(props): number => props.linesPerRow};
-	-webkit-box-orient: vertical;
+	${(props): string =>
```
Comment (Rajat-Dabade) [inline frontend/src/components/Logs/RawLogView/styles.ts:46]: Destruction props at parameter level. 
```suggestion
	${({ isTextOverflowEllipsisDisabled,  linesPerRow }): string =>
```

File: frontend/src/components/Logs/RawLogView/index.tsx [hunk 2]
```diff
@@ -37,12 +37,19 @@ const convert = new Convert();
...
 	isActiveLog?: boolean;
 	isReadOnly?: boolean;
+	isTextOverflowEllipsisDisabled?: boolean;
 	data: ILog;
 	linesPerRow: number;
 }
 
 function RawLogView(props: RawLogViewProps): JSX.Element {
-	const { isActiveLog = false, isReadOnly = false, data, linesPerRow } = props;
+	const {
```
Comment (Rajat-Dabade) [inline frontend/src/components/Logs/RawLogView/index.tsx:46]: As we are refactoring the code, can you please destructure the props at the parameter level? 
```suggestion
	function RawLogView({
		isActiveLog = false,
		isReadOnly = false,
		data,
		linesPerRow,
		isTextOverflowEllipsisDisabled = false
	}: RawLogViewProps): JSX.Element {
```

File: frontend/src/components/Logs/RawLogView/index.tsx [hunk 3]
```diff
@@ -37,12 +37,19 @@ const convert = new Convert();
...
+		isActiveLog = false,
+		isReadOnly = false,
+		data,
+		linesPerRow,
+		isTextOverflowEllipsisDisabled = false,
```
Comment (Rajat-Dabade) [inline frontend/src/components/Logs/RawLogView/index.tsx:51]: No need to provide `isTextOverflowEllipsisDisabled` as `false`, as it is already mentioned as `false` in `RawLogView.defaultProps`.  Also please remove the false from `isActiveLog` and `isReadOnly` props.

File: frontend/src/components/Logs/RawLogView/index.tsx [hunk 4]
```diff
@@ -37,12 +37,19 @@ const convert = new Convert();
 interface RawLogViewProps {
 	isActiveLog?: boolean;
 	isReadOnly?: boolean;
+	isTextOverflowEllipsisDisabled?: boolean;
```
Comment (Rajat-Dabade) [inline frontend/src/components/Logs/RawLogView/index.tsx:40]: As a part of refactoring, can we move this interface to `components/Logs/RawLogView/types.ts`

**Review by Rajat-Dabade — CHANGES_REQUESTED** [review]
Some minor changes, mostly refactoring ones.

**Comment by CKanishka** [comment]
> Some minor changes, mostly refactoring ones.

@Rajat-Dabade  Addressed the comments

**Review by palashgdev — APPROVED** [review]
LGTM

**Review by Rajat-Dabade — APPROVED** [review]
LGTM

**Review by YounixIO — APPROVED** [review]

**Comment by srikanthccv** [comment]
@Rajat-Dabade, please squash when you merge to develop. Otherwise, each WIP commit shows up as in release notes and commit history.

**Pull request merged**